WORKSPACE

```diff
@@ -23,8 +23,6 @@ http_archive(
     url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
 )
 
-load("@rules_jvm_external//:defs.bzl", "maven_install")
-
```

Comment:
Note: This line was a duplicate. You can find the version of this line that I kept on line 106.

Reply:
Ah, thanks for the clarification.